src/serac/numerics/mesh_utils.cpp

```diff
@@ -101,10 +101,12 @@ std::shared_ptr<mfem::ParMesh> buildDiskMesh(int approx_number_of_elements, cons
 
   squish(mesh);
 
-  return std::make_shared<mfem::ParMesh>(comm, mesh);
+  // The copy ctor is marked explicit, but this should qualify for NRVO...
+  // It looks like this (and its analogues) don't have their copies elided...
+  return mfem::Mesh(mesh);
```

Comment:
We should add an issue (with a link to the MFEM PR) so we don't forget to get rid of this extra copy.